recipes/sseclient-py/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "sseclient-py" %}
+{% set version = "1.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sseclient-py-{{ version }}.tar.gz
+  sha256: c547c5c1a7633230a38dc599a21a2dc638f9b5c297286b48b46b935c71fac3e8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - sseclient
```

Comment:
@sanurielf Are you sure this should be here? `sseclient` is a _different_ Python library for SSE: https://github.com/btubbs/sseclient, and `sseclient-py` does not depend on it.

Reply:
Fixed, thanks!